tests/func/test_repro_multistage.py

```diff
@@ -517,37 +517,36 @@ def test_repro_multiple_params(tmp_dir, dvc):
 
 
 def test_repro_list_of_commands_in_order(tmp_dir, dvc):
-    dvcfile_content = {
-        "stages": {
-            "multi_commands": {
-                "cmd": [
-                    'echo "First command" > foo',
-                    'echo "Second command" >> foo',
-                ],
-                "outs": ["foo"],
-            }
-        }
-    }
-    dump_yaml(PIPELINE_FILE, dvcfile_content)
-    dvc.reproduce(target="multi_commands")
-    assert (tmp_dir / "foo").read_text() == "First command\nSecond command\n"
+    (tmp_dir / "dvc.yaml").write_text(
```

Comment:
cool, I was not sure if it was better to write yaml directly or to use the `dump_yaml` func indeed.

Reply:
should the `PIPELINE_FILE` variable be used instead of direct `dvc.yaml` ?

Reply:
Doesn't really matter in the test.